packages/hydrogen/src/build.ts

```diff
@@ -118,6 +120,15 @@ export const build: BuildV2 = async ({
     deploymentTarget: 'v8-worker',
     entrypoint: 'index.js',
     files: edgeFunctionFiles,
+    regions: (() => {
+      try {
+        const project = new Project();
+        const config = getConfig(project, edgeFunctionFiles['index.js'].fsPath);
+        return config?.regions;
+      } catch {
+        return undefined;
```

Comment:
Can we debug log the warning that was here? I imagine trying to debug an issue around regions not working and having nothing that points me to this chunk of code.

Reply:
I can add this in a different PR but tbh hydrogen being a single edge function deployment might not be the best approach for regional invocations? <picture data-single-emoji=":surprised-wide-eyed:" title=":surprised-wide-eyed:"><img class="emoji" width="20" height="auto" src="https://single-emoji.vercel.app/api/emoji/eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..nBLgKyW2RWB7OU4u.KtGdSchSvJncwxnTchZqscpbrSSQkOt2audbsdaIelxshIHtKVbiabRmwQ2v7SO7GtjGAPEuYzi_15p0haNSyxNwsfPoSiZYuhLTkloYtFxajo6OTXTUjnQqd0JMWTM-wldkJFTb.J9AnoaG5vo7ZYuJxL5tlrQ" alt=":surprised-wide-eyed:" align="absmiddle"></picture>